src/main/java/org/opentripplanner/model/Station.java

```diff
@@ -159,6 +159,7 @@ public GeometryCollection getGeometry() {
   }
 
   private static GeometryCollection computeGeometry(WgsCoordinate coordinate, Set<StopLocation> childStops) {
+    if(coordinate == null) { return null; }
```

Comment:
For flex areas, which also don't have a single coordinate, we compute the centroid from the child stations. Could you do that here as well?

Reply:
I have refactored this method, so that it calculates geometry only from child stops location, if stationPoint is null, please have a look.